ui/test/isDateWithin_test.js

```diff
@@ -18,10 +18,10 @@ describe('isDateWithin', () => {
 
   it('should return false', () => {
     [
-      '44 minutes',
+      '42 minutes',
```

Comment:
Any idea what's wrong?  Is this function off by several minutes?

Reply:
I think the `differenceInMinutes` function from date-fns rounds down:

```
> differenceInMinutes(new Date('December 1, 2019 03:25:59'), new Date('December 1, 2019 03:24:00'))
1
```